reporter.js

```diff
@@ -0,0 +1,168 @@
+/**
+ * @typedef {Object} TestResult
+ * @property {Boolean} failed - whether or not test failed
+ * @property {String} name - name of test
+ * @property {Boolean} passed - whether or not test passed
+ * @property {Number} runDuration - run duration in milliseconds
+ * @property {Boolean} skipped - whether or not test was skipped
+ */
+
+/**
+ * Get duration in human friendly format
+ * @param {Number} value - duration in milliseconds
+ * @returns {String} human friendly duration
+ */
+function getHumanReadableDuration (value) {
+  if (value < 1000) {
+    return value + ' ms'
+  }
+
+  // Convert from milliseconds to seconds
+  value = value / 1000
+
+  if (value < 60) {
+    return value + ' s'
+  }
+
+  // Convert from seconds to minutes
+  value = value / 60
+
+  if (value < 60) {
+    return value + ' min'
+  }
+
+  // Convert from minutes to hours
+  value = value / 60
+
+  return value + ' hr'
+}
+
+/**
+ * Sort tests by duration
+ * @param {TestResult} a - first test
+ * @param {TestResult} b - second test
+ * @returns {Number} number indicating sort order
+ */
+function testSorter (a, b) {
+  if (a.runDuration < b.runDuration) {
```

Comment:
I usually do `a.runDuration - b.runDuration` for ordinal sorts. You can actually return ranges > 1 or < 1.